share/jupyter/kernels/xcpp17/kernel.json.in

```diff
@@ -4,7 +4,9 @@
       "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR@/xcpp",
       "-f",
       "{connection_file}",
-      "-std=c++17"
+      "-std=c++17",
+      "-I",
```

Comment:
This might break on other platforms but works perfectly on Windows.

Can someone please test the change on Linux and Mac OS X? I currently don't have access to these platforms.